orttraining/orttraining/python/training/torchdynamo/ort_backend.py

```diff
@@ -16,18 +16,47 @@
 import torch.fx
 import torch.jit
 import torch.onnx
+import torch.onnx._internal
```

Comment:
Maybe not. There are different kinds of `_internal` spreading in the code. If we label it this way, we need to label A LOT of lines with this comment. It will look funny.

Reply:
The FIX will automatically happen since otherwise the code throws.

Reply:
I just meant this blob since they seem to be clustered together here in the import section. We also don't want readers to stumble on this code and think importing from _internal is the right way to develop with torch.onnx. I was going to propose all caps "THIS IS DANGEROUS" sort of thing in the comment but at least use https://github.com/microsoft/onnxruntime/issues/11901 to track?